server/api/crud/model_monitoring/deployment.py

```diff
@@ -869,10 +868,27 @@ async def delete_monitoring_function(
             auth_info=auth_info,
             background_task_name=background_task_name,
         )
-        if delete_v3io_stream:
-            import v3io.dataplane
-            import v3io.dataplane.response
+        if delete_app_stream_resources:
+            MonitoringDeployment._delete_model_monitoring_stream_resources(
+                project=project,
```

Comment:
You're inside this class - I would make `_delete_model_monitoring_stream_resources` a standard method and won't pass `project` from method to method, it's in the "self" object.

Edit - after a second thought, I understand this design comes from the background task process.